spec/i18n_spec.rb

```diff
@@ -16,9 +16,42 @@ def translate_expectation(binding, inspection)
+    it { expect(translate_expectation('foo', 'HasBinding')).to eq('<strong>foo</strong> debe estar declarado') }
+
+    it { expect(translate_expectation('foo.bar', 'HasMethod')).to eq('el método <strong>foo.bar</strong> debe estar declarado') }
+    it { expect(translate_expectation('foo.bar', 'HasIf')).to eq('<strong>foo.bar</strong> debe usar if') }
+
+    it { expect(translate_expectation('Intransitive:foo', 'Not:HasLambda')).to eq('<strong>foo</strong> no debe emplear expresiones lambda') }
+
+    it { expect(translate_expectation('foo', 'HasUsage:=baz')).to eq('<strong>foo</strong> debe utilizar <strong>baz</strong>') }
+    it { expect(translate_expectation('foo', 'HasUsage:~baz')).to eq('<strong>foo</strong> debe delegar en algo parecido a <strong>baz</strong>') }
+    it { expect(translate_expectation('foo', 'HasMethod:~baz')).to eq('<strong>foo</strong> debe declarar un método parecido a <strong>baz</strong>') }
+    it { expect(translate_expectation('foo', 'HasUsage:*')).to eq('<strong>foo</strong> debe ser utilizado') }
+    it { expect(translate_expectation('foo', 'HasMethod:*')).to eq('el método <strong>foo</strong> debe estar declarado') }
+    it { expect(translate_expectation('foo', 'HasUsage:baz')).to eq('<strong>foo</strong> debe utilizar <strong>baz</strong>') }
+    it { expect(translate_expectation('foo', 'HasUsage')).to eq('<strong>foo</strong> debe ser utilizado') }
+    it { expect(translate_expectation('foo', 'HasForall')).to eq('<strong>foo</strong> debe utilizar forall') }
+
+    it { expect(translate_expectation('foo', 'Not:HasUsage:=baz')).to eq('<strong>foo</strong> no debe utilizar <strong>baz</strong>') }
+    it { expect(translate_expectation('foo', 'Not:HasUsage:~baz')).to eq('<strong>foo</strong> no debe delegar en algo parecido a <strong>baz</strong>') }
+    it { expect(translate_expectation('foo', 'Not:HasUsage:*')).to eq('<strong>foo</strong> no debe ser utilizado') }
+    it { expect(translate_expectation('foo', 'Not:HasUsage')).to eq('<strong>foo</strong> no debe ser utilizado') }
     it { expect(translate_expectation('foo', 'Not:HasUsage:baz')).to eq('<strong>foo</strong> no debe utilizar <strong>baz</strong>') }
     it { expect(translate_expectation('foo', 'Not:HasLambda')).to eq('<strong>foo</strong> no debe emplear expresiones lambda') }
+
+    it { expect(translate_expectation('', 'HasClass')).to eq('la solución debe declarar clases') }
+    it { expect(translate_expectation('', 'Not:HasMethod')).to eq('la solución no debe declarar métodos') }
+    it { expect(translate_expectation('', 'Not:HasClass')).to eq('la solución no debe declarar clases') }
+
+    it { expect(translate_expectation('foo', 'HasObject:^')).to eq('<strong>foo</strong> debe declarar objetos') }
+    it { expect(translate_expectation('', 'Not:HasClass')).to eq('la solución no debe declarar clases') }
```

Comment:
Correct me if I'm wrong;

'foo', 'HasUsage:=baz' now would mean that foo should use baz exactly.
'foo', 'HasUsage:~baz' now would mean that foo should use something similar to baz.

Using that logic, I think I'd prefer
'foo', 'HasUsage:~*' to mean that foo should use something, anything.
I'd say 'HasUsage:=*' produces the same effect.

Other topic: right now, however, I find myself questioning why 'foo', 'HasUsage:*' means that foo must be used by anyone. The semantic of "foo" inverts in this case. Couldn't we have a * in the binding function name? 
'*', 'HasUsage:=foo' 
(meaning "foo must be used somewhere")

I'm a bit rusty on Mumuki exercise definition, so sorry If I mixed stuff up.